plugins/upstash/upstash.go

```diff
@@ -0,0 +1,25 @@
+package upstash
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func UpstashCLI() schema.Executable {
+	return schema.Executable{
+		Name:      "Upstash CLI", 
+		Runs:      []string{"upstash"},
+		DocsURL:   sdk.URL("https://github.com/upstash/cli"), 
+		NeedsAuth: needsauth.IfAll(
```

Comment:
Let's exclude `upstash auth` as well. Also, if there's any other one that should be in here, do add it 😄

Reply:
Resolving this because a new line was added per this comment. However, continuing this discussion in comment: https://github.com/1Password/shell-plugins/pull/316/files#r1243472288